lib/discount.rb

```diff
@@ -0,0 +1,31 @@
+require "./lib/modules/percentage"
+
+class Discount 
+  attr_reader :discount, :description
+  include Percentage
+
+  def initialize options
+    @discount               = options[:discount]
```

Comment:
You could use named parameters here which is a slightly cleaner way of doing this initializer, although this is fine